Jellyfin.Plugin.Tvdb/TvdbClientManager.cs

```diff
@@ -393,7 +491,30 @@ public async Task<IReadOnlyList<ArtworkType>> GetArtworkTypeAsync(CancellationTo
         }
         else
         {
-            return seriesData.Episodes[0].Id?.ToString(CultureInfo.InvariantCulture);
+            var tvdbId = seriesData.Episodes[0].Id?.ToString(CultureInfo.InvariantCulture);
+            if (key != null)
+            {
+                _memoryCache.Set(key, tvdbId, TimeSpan.FromHours(CacheDurationInHours));
+            }
+
+            return tvdbId;
+        }
+    }
+
+    /// <summary>
+    /// Purge the cache.
+    /// </summary>
+    /// <returns>True if success else false.</returns>
+    public bool PurgeCache()
+    {
+        if (_memoryCache is MemoryCache memoryCache)
+        {
+            memoryCache.Compact(1);
```

Comment:
IMemoryCache is a singleton. You're nuking the cache for the entire server.

Reply:
Reverted

Reply:
You can just `new` your own memorycache if you want to keep the functionality